### PR TITLE
pkp/pkp-lib#990 Fix positioning of tinymce tooltips in modals

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

File: styles/form/form.less [hunk 1]
```diff
@@ -466,6 +466,16 @@
 	cursor: pointer;
 }
 
+// Fixes the positioning of tinymce tooltips
+// skin.min.css (the tinymce we're loading) sets their position to absolute,
+// but the tooltips seem to set position relative to the viewport, not the
+// document. This causes problems for textareas in modals, where the scroll
+// position of the modal differs from the scroll position of the document.
+// "I'm almost definitely pretty sure that might be the explanation." ~NW
```
Comment (beghelli) [inline styles/form/form.less]: Trying to convincing yourself? :P

**Comment by beghelli** [comment]
@NateWr I don't think this can impact the tests right?

**Comment by NateWr** [comment]
Nope. Should be a clean commit.

File: styles/form/form.less (hunk 1 above)
Comment (NateWr) [inline styles/form/form.less]: Sometimes we find the answer without ever really knowing the question. :)

**Pull request merged**